src/EventStore.Core/Certificates/OptionsCertificateProvider.cs

```diff
@@ -6,12 +6,8 @@ namespace EventStore.Core.Certificates {
 	public class OptionsCertificateProvider: CertificateProvider {
 		private static readonly ILogger Log = Serilog.Log.ForContext<ClusterVNode>();
 		private ClusterVNodeOptions _options;
```

Comment:
One last change, don't think there's a need for this anymore 🤔 

Besides that it looks good 👍

Reply:
Updated